src/index.tsx

```diff
@@ -18,4 +25,12 @@ app.get("/", (c) => {
 	);
 });
 
+app.get("/board", async (c) => {
+	const vars = env<Env>(c);
+	const redis = Redis.fromEnv(vars);
```

Comment:
keep in mind, if we have more environment variables in the future, we'd be passing _all_ of them to this method. it may not be very secure, but typescript may also yell at us if we handle it correctly anyways.

could do something like this
```tsx
	const env = adapter.env<Env>(c, "workerd");
	const redis = Redis.fromEnv({
		UPSTASH_REDIS_REST_URL: env.UPSTASH_REDIS_REST_URL,
		UPSTASH_REDIS_REST_TOKEN: env.UPSTASH_REDIS_REST_TOKEN,
	});
```